### PR TITLE
feat: Include list of image layer directories in `ContainerInfo`

### PR DESCRIPTION
**Pull request by vadorovsky**
For each container, try to retrieve information about image layers paths (from the host filesystem's perspective) and include them in `ContainerInfo`.

### CODE REVIEW

**Review by banditopazzo — CHANGES_REQUESTED** [review]
Please handle the `unwrap`s

**Review by banditopazzo — CHANGES_REQUESTED** [review]
In general it seems fine, but there is no support for podman

File: crates/bpf-common/src/containers/mod.rs [hunk 1]
```diff
@@ -266,6 +307,8 @@ impl ContainerInfo {
             name: config.name,
             image: config.rootfs_image_name,
             image_digest: image.digest.clone(),
+            // TODO(vadorovsky): Parse layer information in Podman.
```
[the remark this answers is not in the document]
Comment (vadorovsky) [reply crates/bpf-common/src/containers/mod.rs:310]: Added in #309 309

File: crates/bpf-common/src/containers/mod.rs [hunk 2]
```diff
@@ -266,6 +307,8 @@ impl ContainerInfo {
...
+            layers: Vec::new(),
```
[the remark this answers is not in the document]
Comment (vadorovsky) [reply crates/bpf-common/src/containers/mod.rs]: I must've done a wrong `git push -f`. I restored them now, sorry.

**Review by banditopazzo — APPROVED** [review]

**Pull request merged**